Gini-iOS-SDK/GINIDocumentTaskManager.m

```diff
@@ -72,21 +72,29 @@ - (BFTask *)getDocumentWithId:(NSString *)documentId{
 }
 
 - (BFTask *)createDocumentWithFilename:(NSString *)fileName fromImage:(UIImage *)image {
-    NSParameterAssert([fileName isKindOfClass:[NSString class]]);
-    NSParameterAssert([image isKindOfClass:[UIImage class]]);
+    return [self createDocumentWithFilename:fileName fromImageData:UIImageJPEGRepresentation(image, 0.2)];
+}
+
+- (BFTask *)createDocumentWithFilename:(NSString *)fileName fromImage:(UIImage *)image docType:(NSString *)docType {
+    return [self createDocumentWithFilename:fileName fromImageData:UIImageJPEGRepresentation(image, 0.2) docType:docType];
+}
 
-    BFTask *createTask = [[_apiManager uploadDocumentWithData:UIImageJPEGRepresentation(image, 0.2) contentType:@"image/jpeg" fileName:fileName docType:nil] continueWithSuccessBlock:^id(BFTask *task) {
+- (BFTask *)createDocumentWithFilename:(NSString *)fileName fromImageData:(NSData *)imageData {
```

Comment:
This could call [self createDocumentWithFilename:fromImageData:doctype:nil] and the called method could allow nil as doctype. Then only one method calls the api communicator.